ee/ucp/kubernetes/storage/use-azure-files.md

```diff
@@ -238,6 +240,48 @@ spec:
 EOF
 ```
 
+### Troubleshooting
+
+When creating Persistent Volume Claims the volume may constantly stay in a
+`Pending` state.
+
+```
+$ kubectl get pvc
+NAME             STATUS    VOLUME   CAPACITY   ACCESS MODES   STORAGECLASS   AGE
+azure-file-pvc   Pending                                      standard       32s
+```
+
+If that is the case, the `persistent-volume-binder` service account does not
+have the relevant Kubernetes RBAC permissions. The storage account creates a
+Kubernetes secret to store the Azure Files Storage Account Key.
+
+```
+Warning    ProvisioningFailed  7s (x3 over 37s)  persistentvolume-controller
+Failed to provision volume with StorageClass "standard": Couldn't create secret
+secrets is forbidden: User "system:serviceaccount:kube-system:persistent-volume-binder"
+cannot create resource "secrets" in API group "" in the namespace "default": access denied
+```
+
+To grant the `persistent-volume-binder` service account the relevant the RBAC
```

Comment:
Suggestion: "To grant the `persistent-volume-binder` service account, the relevant the RBAC..."